package.json

```diff
@@ -17,7 +17,9 @@
     "prepare": "npm run compile",
     "pretest": "npm run compile",
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
-    "commitlint": "git log -1 --pretty=%B | commitlint"
+    "commitlint": "git log -1 --pretty=%B | commitlint",
+    "webpack": "webpack",
+    "browser-test": "sh browser-test/browser-test.sh"
```

Comment:
Would be rad if we could just rely on mocha here (don't see why we couldn't)

Reply:
Karma tests are run by a connected browser (Chromium). We cannot start a web server in browser, and we need a web server to test fetch (and we need it running locally to make tests hermetic - I can as well query `https://cnn.com` but I don't feel it's correct). So we need both Node.js process that runs `express`, and browser that runs tests. Hence the script.

(I can make the same script in TypeScript to avoid using shell)

Reply:
This is more a nit than anything.  Whatever you think is best :)

Reply:
Wrote that in TypeScript. It's even cleaner now since the webserver and the test runner are in the same script.